src/components/Navigation/Navigation.jsx

```diff
@@ -37,32 +37,22 @@ export default function Navigation(){
 		}
 	];
 
-
-	const links       = linkData.map(RENDER.item);
-	const ariaOpen    = open.toString();
+	const links      = linkData.map(RENDER.item);
+	const expanded   = open.toString();
```

Comment:
what conclusion did we reach with `toString()` was it needed in the end, or was it just with vanilla javascript that you need to stringify it?

Reply:
Yarp, works fine without toString :)